pkgs/applications/misc/solaar/default.nix

```diff
@@ -0,0 +1,36 @@
+{ fetchurl, stdenv, makeWrapper, python3Packages , gtk3 }:
+python3Packages.buildPythonPackage rec {
+  name = "solaar-${version}";
+  version = "0.9.4";
+  src = fetchurl {
+    sha256 = "0954grz2adggfzcj4df4mpr4d7qyl7w8rb4j2s0f9ymawl92i05j";
+    url = "https://github.com/pwr/Solaar/archive/0.9.2.tar.gz";
+  };
+
+  buildInputs =  [ gtk3 python3Packages.pygobject3 python3Packages.pyudev ];
+  enableParallelBuilding = true;
+  postInstall = ''
+    wrapProgram "$out/bin/solaar" \
+      --prefix PYTHONPATH : "$PYTHONPATH" \
+      --prefix GI_TYPELIB_PATH : "$GI_TYPELIB_PATH"
+    wrapProgram "$out/bin/solaar-cli" \
+      --prefix PYTHONPATH : "$PYTHONPATH" \
+      --prefix GI_TYPELIB_PATH : "$GI_TYPELIB_PATH"
+  '';
+  meta = with stdenv.lib; {
+    description = "Linux devices manager for the Logitech Unifying Receiver.";
```

Comment:
I know there are guidelines, I was asking is there any script to validate stuff. Most of the "package" is validated because of nix language and internal structures but as it looks like there is few thinks which can be improved. Maybe we could add a check to meta field (or make it a function, I'm pretty new and not sure what would be the best)?